src/components/custom-field-input-number/custom-field-input-number.test.jsx

```diff
@@ -157,9 +157,17 @@ describe('CustomFieldInputNumber', () => {
 
   describe('value API', () => {
     it('is valid on a positive integer', () => {
-      render(<CustomFieldInputNumber {...requiredProps} value={1} />);
+      const ref = createRef();
+      render(<CustomFieldInputNumber ref={ref} {...requiredProps} value={1} />);
```

Comment:
This is an observation: the order of the supplied props matter!

In this case, it is usually best to spread the `requiredProps` before any test specific props. The test-specific props will clobber any props in `requiredProps`.

While this is technically not wrong, it could cause headaches down the road. (Same below)